example/src/main/java/io/honeycomb/opentelemetry/android/example/ViewInstrumentationPlayground.kt

```diff
@@ -0,0 +1,125 @@
+
+@Composable
+private fun NestedExpensiveView(delayMs: Long) {
+    Row {
+        HoneycombInstrumentedComposable("nested expensive text", LocalOtelComposition.current!!.openTelemetry) {
+            Text(text = timeConsumingCalculation(delayMs))
+        }
+    }
+}
+
+@Composable
+private fun DelayedSlider(
+    delay: Long,
+    onValueChange: (Long) -> Unit,
+) {
+    val (sliderDelay, setSliderDelay) = remember { mutableFloatStateOf(delay.toFloat()) }
+    Slider(
+        value = sliderDelay,
+        onValueChange = setSliderDelay,
+        onValueChangeFinished = { onValueChange(sliderDelay.toLong()) },
+        valueRange = 0f..4000f,
+        steps = 7,
+    )
+}
+
+@Composable
+private fun ExpensiveView() {
+    val (delay, setDelay) = remember { mutableLongStateOf(1000L) }
+
+    HoneycombInstrumentedComposable("main view", LocalOtelComposition.current!!.openTelemetry) {
+        Column(
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+            DelayedSlider(delay = delay, onValueChange = setDelay)
+
+            HoneycombInstrumentedComposable("expensive text 1", LocalOtelComposition.current!!.openTelemetry) {
+                Text(text = timeConsumingCalculation(delay))
+            }
+
+            HoneycombInstrumentedComposable("expensive text 2", LocalOtelComposition.current!!.openTelemetry) {
+                Text(text = timeConsumingCalculation(delay))
+            }
+
+            HoneycombInstrumentedComposable("expensive text 3", LocalOtelComposition.current!!.openTelemetry) {
+                Text(text = timeConsumingCalculation(delay))
+            }
+
+            HoneycombInstrumentedComposable("nested expensive view", LocalOtelComposition.current!!.openTelemetry) {
+                NestedExpensiveView(delayMs = delay)
+            }
+
+            HoneycombInstrumentedComposable("expensive text 4", LocalOtelComposition.current!!.openTelemetry) {
+                Text(text = timeConsumingCalculation(delay))
+            }
+        }
+    }
+}
+
+@Composable
+internal fun ViewInstrumentationPlayground() {
+    val (enabled, setEnabled) = remember { mutableStateOf(false) }
+
+    Column(
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxWidth(),
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+            Text(text = "enable slow render")
+            Switch(
+                checked = enabled,
+                onCheckedChange = setEnabled,
+                modifier = Modifier.testTag("slow_render_switch"),
+            )
+        }
+        if (enabled) {
+            ExpensiveView()
+        }
+    }
+}
```

Comment:
This structure mirrors the same playground view in iOS: https://github.com/honeycombio/honeycomb-opentelemetry-swift/pull/20/files#diff-3ed6499a52b381eba3646ace172cde5bfa2464d33360b412f51b80b35d120c2bR85-R100